server/plugins/renderer/renderer.module.js

```diff
@@ -369,10 +362,10 @@ internals.getTemplatePath = (requestPath, data) => {
 
         if (templatePath) {
             templatePath = path.join('pages/custom', pageType, templatePath.replace(/\.html$/, ''));
+            // eslint-disable-next-line no-param-reassign
+            data.template_file = templatePath;
```

Comment:
Or maybe do you to return here the value and leave the name as it is and we exit iterating over the layouts

Reply:
It's better to rename as we need updated bcapp data earlier than just template path.